backend/src/apiserver/main.go

```diff
@@ -89,25 +132,14 @@ func main() {
 		}
 	}
 
-	logLevel := *logLevelFlag
-	if logLevel == "" {
-		logLevel = "info"
-	}
-
-	level, err := log.ParseLevel(logLevel)
```

Comment:
We still need this because not everything uses `zap` for logging like controller-runtime does.